perf-dash.k8s.io/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+  - bartsmykla
+  - mm4tt
+  - wojtek-t
```

Comment:
nit: add end line?
[same for couple files below]

Reply:
Done